### PR TITLE
Add select CLI

### LINKED ISSUE #26: Add `select` CLI
Add select command where you can choose which `rc` you want to use

### PR DESCRIPTION
**Pull request by mstefferson**
# Context

Added select CLI command and ability to find files. While adding some tests, I fixed some issues with the fixtures

# Changes

* Add method to RCdirs
* Add new CLI command `select`

# Behaves Differently

* Can import fixtures

# Untested

* select CLI not tested.

Closes #26 .

### CODE REVIEW

File: rc4me/run.py [hunk 1]
```diff
@@ -96,5 +97,25 @@ def reset(ctx: Dict[str, RcDirs]):
...
+def select(ctx: Dict[str, RcDirs]):
+    """Swap rc4me configurations
+
+    Displays all available repos and allow user to select one
+    """
+    # Init rc4me directory variables
+    rc_dirs = ctx.obj["rc_dirs"]
+    my_repos = rc_dirs.get_rc_repos()
+    # Show all dirs that aren't curr/prev
+    title = "Please select the run commands repo you want to use: "
```
Comment (jeffmm) [inline rc4me/run.py]: This command is a bit confusing: maybe "Please select the repo/configuration you want to use:" ?

File: rc4me/run.py [hunk 2]
```diff
@@ -96,5 +97,25 @@ def reset(ctx: Dict[str, RcDirs]):
...
+    options = list(my_repos.keys())
+    selected, _ = pick(options, title)
+    logger.info(f"Selected and applying: {my_repos[selected]}")
+    rc_dirs.change_current_to_target(my_repos[selected])
```
Comment (jeffmm) [inline rc4me/run.py]: This will change with my PR but it's fine for now.

File: rc4me/run.py [hunk 3]
```diff
@@ -96,5 +97,25 @@ def reset(ctx: Dict[str, RcDirs]):
...
+    # TODO, should just just be part of change_current_to_target?
```
Comment (jeffmm) [inline rc4me/run.py:117]: Yeah, I agree

**Review by jeffmm — APPROVED** [review]
lgtm

File: rc4me/run.py (hunk 1 above)
Comment (mstefferson) [reply rc4me/run.py]: 👍

File: rc4me/run.py (hunk 2 above)
Comment (mstefferson) [reply rc4me/run.py]: Fixed

File: rc4me/run.py (hunk 3 above)
Comment (mstefferson) [reply rc4me/run.py:117]: On it.

**Pull request merged**